cotainr/cli.py

```diff
@@ -129,11 +129,11 @@ def __init__(
         )
         self.image_path = Path(image_path).resolve()
         if self.image_path.exists():
-            val = input(
+            answer = util.answer_is_yes(
                 f"{self.image_path} already exists. "
-                "Would you like to overwrite it? (y/N) "
-            ).lower()
-            if val != "y":
+                "Would you like to overwrite it? (yes/[N]o) "
```

Comment:
Since `answer_is_yes()` is now responsible for the exact accepted format of the answer, I think the caller shouldn't need to put these in the prompt either.

IOW, `util.answer_is_yes("Is this okay?")` would prompt

```
Is this okay? (yes/[N]o)
```

This also tracks with how that function now changes the prompt after the first query (and includes `Please answer yes/[N]o` internally).

Reply:
In addition to this, we need to keep in mind that with the acceptance of the Miniforge license (for which we also use this new function), we just display the Miniforge license text as-is, which includes the final line `Do you accept the license terms? [yes|no]`. One way or the other we need to make this consistent.

Reply:
Done.
answer_is_yes now adds a yes/[N]o prompt to all inputs and i remove the license [yes|no].